central/processlisteningonport/store/postgres/full_store.go

```diff
@@ -180,6 +245,11 @@ func (s *fullStoreImpl) readRows(
 			ImageId:            procMsg.GetImageId(),
 		}
 
+		if ok, err := plopSAC.ReadAllowed(ctx, sac.ClusterScopeKey(plop.ClusterId), sac.NamespaceScopeKey(plop.Namespace)); err != nil {
```

Comment:
Returns an error if the user does not have access to the cluster and namespace. An alternative is just to filter out such cases. Currently the clusterid and namespace should be the same for all rows, given that they all have the same deploymentid. Thus, it might make sense just to check the first row. However, in the future it is possible that rows could contain rows with multiple different deployments, namespaces, and clusters. It might be better to check each row to be on the safe side.

Reply:
Done